### PR TITLE
Rs/ch/enumerations as classes

### PR DESCRIPTION
**Pull request by charliepnnl**
1. Model the enumerations with a class instead of Enum. This allows us to avoid the need to append `.name` to the enumeration when it is used.
2. Updated all the files that use the enumerations.
3. Removed unused imports of `schema_enum`.
4. Fixed `assert` syntax in `jasonpath_utils.py`.

### CODE REVIEW

File: rct229/data/schema_enums.py [hunk 1]
```diff
@@ -45,10 +63,10 @@ def print_schema_enums():
...
-        for e in schema_enums[key]:
-            print(f"    {e.name}: {e.value}")
+        for e in schema_enums[key].get_list():
+            print(f"    {e}")
         print()
 
 
 # Uncomment this for checking the enumerations after a schema change
-# print_schema_enums()
+print_schema_enums()
```
Comment (weilixu) [inline rct229/data/schema_enums.py:72]: Great solution. Thanks for working on it. 
One comment, should this line be commented out?

**Review by weilixu — APPROVED** [review]
Approve with a minor comment

**Review by yunjoonjung-PNNL — APPROVED** [review]
I aprrreciate this PR, Charlie! I went through your code and it looks perfect to me!

File: rct229/data/schema_enums.py (hunk 1 above)
Comment (charliepnnl) [reply rct229/data/schema_enums.py]: Good catch!  Done.

**Pull request merged**